share/completions/istioctl.fish

```diff
@@ -0,0 +1,176 @@
+# fish completion for istioctl                             -*- shell-script -*-
+
+function __istioctl_debug
+    set -l file "$BASH_COMP_DEBUG_FILE"
+    if test -n "$file"
+        echo "$argv" >> $file
+    end
+end
+
+function __istioctl_perform_completion
+    __istioctl_debug "Starting __istioctl_perform_completion"
```

Comment:
Please remove the debugging stuff. It's not of interest to us.

Reply:
done